CONTRIBUTING.md

```diff
@@ -6,12 +6,13 @@ see [BUILDING.md](BUILDING.md)
 
 ## Next steps
 * Make your changes/add your feature/fix a bug.
-* Test your feature branch changes
-* Check your formatting
-* Submit a pull request
+* Test your feature branch changes.
+* Check your formatting.
+* Submit a pull request.
 
 ## Testing
-Before submitting a pull request the unit and integration tests must all pass. We are using CTest, (Please see [the CTest documentation](https://cmake.org/Wiki/CMake/Testing_With_CTest) for further information.)
+Before submitting a pull request the unit and integration tests must all pass. We are using CTest (please see [the CTest documentation](https://cmake.org/Wiki/CMake/Testing_With_CTest) for further information) for unit tests and old integration tests, and [Google Test](https://github.com/google/googletest) for new integration tests.
```

Comment:
We actually only use CTest to run tests. CTest itself is not a testing framework but a runtime management thing. All tests, including new and old use CTest for running. You can however run the unit and new integration tests, which are written in gtest, directly, like from your IDE. Running the old unit tests, which are written in a home grown test framework similar to the main java project's DUnit, is a lot more complicated from the IDE.